src/main/java/io/jenkins/plugins/generate/parsers/RootPluginDataParser.java

```diff
@@ -27,6 +28,8 @@ public void parse(JSONObject pluginJson, Plugin plugin) {
     plugin.setSha1(pluginJson.optString("sha1", null));
     plugin.setTitle(pluginJson.optString("title", null));
     plugin.setUrl(pluginJson.optString("url", null));
+    JSONObject issueTracker = (JSONObject) pluginJson.opt("issueTracker");
+    plugin.setIssueTracker(new IssueTracker(issueTracker.getString("name"), issueTracker.getString("url")));
```

Comment:
You get issue tracker optionally, then use it immediately as if it's not null.

You need to confirm its not null before getting

Reply:
I think it's also causing the build to fail.

I'd also make sure there's a test that shows the data is loaded properly

Reply:
Update site will always set it after my change so it’s not really optional anymore, but may as well make it optional for safety I guess

Will take a look at the tests later